src/identityProvider/IdentityProviderHandler.ts

```diff
@@ -0,0 +1,24 @@
+import { getLoggerFor } from '../logging/LogUtil';
```

Comment:
Naming convention: the folder name is `identityProvider` but we probably want `idp` or similar (following the other names).

Reply:
I went with `identityProvider` because `idp` looks too much like `ldp`, but I can still swap it over if you prefer `idp`.

Reply:
Hah good point. Or `identity`. But not camelCase.